syntax = "proto3";

package qms;

option java_package = "com.nevmem.qms";
option java_outer_classname = "OrganizitionProto";

message OrganizationInfo {
  string id = 1;
  string name = 2;
  string address = 3;
}
